spec.html

```diff
@@ -32103,6 +32115,55 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          ECMAScript identifies time zones using the Zone and Link names of the IANA Time Zone Database <a href="https://www.iana.org/time-zones/">https://www.iana.org/time-zones/</a>.
```

Comment:
Given that the IANA database calls them 'names', would it make more sense for ECMAScript to call them 'names' too, rather than 'identifiers'?

Reply:
IANA does use "name" primarily, but apparently they're not averse to "identifiers" either, per this heading from https://data.iana.org/time-zones/theory.html#naming. 

<img width="622" alt="image" src="https://user-images.githubusercontent.com/277214/229980256-7cdc7f25-086c-4f07-87a3-44bfd85f6999.png">

We decided to avoid "time zone name" to avoid ambiguity with the existing `timeZoneName` option of `Intl.DateTimeFormat`, which controls the output of the localized name of the time zone, not the IANA name:

```js
new Intl.DateTimeFormat('en', { timeZoneName: 'short' }).format(new Date());
// => '4/4/2023, PDT'
```

We chose "identifier" because it aligns with terms used for similar things in ECMA-402, like [Numbering System Identifiers](https://tc39.es/ecma402/#sec-numberingsystem-identifiers), [Unicode Locale Identifiers](https://tc39.es/ecma402/#sec-Intl.Locale), [Measurement Unit Identifiers](https://tc39.es/ecma402/#sec-measurement-unit-identifiers), etc.

Also because it aligns with usage in Temporal, both internal (e.g. [Temporal.TimeZone's `[[Identifier]]` internal slot ](https://tc39.es/proposal-temporal/#sec-temporal-createtemporaltimezone) as well as the public API like [`Temporal.TimeZone.prototype.id`](https://tc39.es/proposal-temporal/#sec-get-temporal.timezone.prototype.id).

We actually used to call the public API getter `name` instead of `id` but we ended up changing it to `id` for the reasons noted above. I can dig up the notes from the meeting where we made the change, if you're interested in the details.

The general guideline we've tried to follow in Temporal and 402 is that "name" is a localized string meant for end-users to read, while "identifier" (or "id" for short) is a computer-readable string meant for programs to interpret.

So if OK with you, we'd prefer to keep "identifier" here. Let me know if that's a problem and we can discuss further.

If this PR is merged, then the follow-up PRs for Temporal and 402 will ensure we're using the same "identifier" term throughout. Today there's some inconsistency in both specs.